adapters/Unity/UnityAdapter/GADMAdapterUnityRewardedAd.m

```diff
@@ -67,27 +70,27 @@ - (void)requestRewardedAd {
   }
   
   if (![UnityAds isInitialized]) {
-    [[GADMAdapterUnity alloc] initializeWithGameID:_gameID withInitDelegate:Nil];
+    [[GADMAdapterUnity alloc] initializeWithGameID:_gameID withCompletionHandler:nil];
   }
 
-  @synchronized (_placementInUseRewarded) {
+  dispatch_sync(_lockQueue, ^{
```

Comment:
This sync is too long. If the completion handler invokes code that does another sync on this _lockQueue, then we're blocked.